Assets/Mirage/Editor/WelcomeWindow/Resources/WelcomeWindow.uxml

```diff
@@ -2,7 +2,7 @@
     <Style src="WelcomeWindow.uss" />
     <ui:VisualElement name="Banner">
         <ui:Label name="VersionText" text="Placeholder" />
-        <ui:Image name="Icon" style="background-image: url(&apos;/Assets/Mirage/Editor/WelcomeWindow/Resources/MirageIcon.png&apos;);" />
+        <ui:Image name="Icon" style="background-image: url(&apos;/Packages/com.miragenet.mirage/Editor/WelcomeWindow/Resources/MirageIcon.png&apos;);" />
```

Comment:
It is in the resources folder to solve this situation regardless of how you install Mirage.
Try this:
```suggestion
        <ui:Image name="Icon" style="background-image: resource('MirageIcon.png');" />
```